app/src/androidTest/java/com/example/bigowlapp/systemTests/InviteByPhoneTest.java

```diff
@@ -0,0 +1,206 @@
+package com.example.bigowlapp.systemTests;
+
+
+import android.os.SystemClock;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+
+import androidx.test.espresso.DataInteraction;
+import androidx.test.espresso.ViewInteraction;
+import androidx.test.filters.LargeTest;
+import androidx.test.rule.ActivityTestRule;
+import androidx.test.runner.AndroidJUnit4;
+
+import com.example.bigowlapp.R;
+import com.example.bigowlapp.activity.WelcomePageActivity;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+import org.junit.Ignore;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import static androidx.test.espresso.Espresso.onData;
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static androidx.test.espresso.action.ViewActions.replaceText;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withClassName;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+import static androidx.test.espresso.matcher.ViewMatchers.withText;
+import static org.hamcrest.Matchers.allOf;
+import static org.hamcrest.Matchers.anything;
+import static org.hamcrest.Matchers.is;
+
+@Ignore("Not to run on the CI")
+@LargeTest
+@RunWith(AndroidJUnit4.class)
+
+
+public class InviteByPhoneTest {
```

Comment:
Just remove the  spaces in the section (line 42 and 43).

It still works but the annotation are directly applied to the class, so the whitespace is kind of misleading.